Missing lwt_ssl in nix workflow

I want to experiment with nix cross-compilation as @anmonteiro  recommended in [discuss](https://discuss.ocaml.org/t/cross-compiling-implementations-how-they-work/8686/17 ).
I created [my own repo](https://github.com/Kakadu/test-nix-ocaml/) with the same contents as this repo but Nix build is unable to download 'https://github.com/anmonteiro/lwt_ssl/archive/e68d8aab.tar.gz' and fails. There is a chance, that if you rerun your actions in this repo, they will fail too. 

How should I get working nix-based cross compilation?

[full log](https://github.com/Kakadu/test-nix-ocaml/actions/runs/6024767254/job/16344135144): 

```
Run nix build -L .#arm64-musl
  nix build -L .#arm64-musl
  shell: /usr/bin/bash -e {0}


copying path '/nix/store/y9[2](https://github.com/Kakadu/test-nix-ocaml/actions/runs/6024767254/job/16344135144#step:4:2)sva1921n1vczhqidc9nljfnsp2wyy-source' from 'https://cache.nixos.org'...
error:
       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       … while evaluating derivation 'ocaml4.14.0-doi2bib-n-a-aarch64-unknown-linux-musl'
         whose name attribute is located at /nix/store/y92sva1921n1vczhqidc9nljfnsp2wyy-source/pkgs/stdenv/generic/make-derivation.nix:270:7

       … while evaluating attribute 'OCAMLFIND_CONF' of derivation 'ocaml4.14.0-doi2bib-n-a-aarch64-unknown-linux-musl'

         at /nix/store/rljyx9nvvi9c662c8d8mvvdd4ys60q8j-source/cross/ocaml.nix:140:1[3](https://github.com/Kakadu/test-nix-ocaml/actions/runs/6024767254/job/16344135144#step:4:3):

          139|
          1[4](https://github.com/Kakadu/test-nix-ocaml/actions/runs/6024767254/job/16344135144#step:4:5)0|             OCAMLFIND_CONF = oself.makeFindlibConf args;
             |             ^
          141|             buildPhase = ''

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: unable to download 'https://github.com/anmonteiro/lwt_ssl/archive/e[6](https://github.com/Kakadu/test-nix-ocaml/actions/runs/6024767254/job/16344135144#step:4:7)[8](https://github.com/Kakadu/test-nix-ocaml/actions/runs/6024767254/job/16344135144#step:4:9)d8aab.tar.gz': HTTP error 404

       response body:

       Not Found
Error: Process completed with exit code 1.
```

--- a/flake.nix
+++ b/flake.nix
@@ -12,12 +12,14 @@
         pkgs = nixpkgs.legacyPackages."${system}".extend (self: super: {
           ocamlPackages = super.ocaml-ng.ocamlPackages_4_14;
         });
+        native = pkgs.callPackage ./nix {
+          nix-filter = nix-filter.lib;
+        };
+
       in
       {
         packages = {
-          native = pkgs.callPackage ./nix {
-            nix-filter = nix-filter.lib;
-          };
+          inherit native;
           musl =
             let
               pkgs' = pkgs.pkgsCross.musl64;
@@ -30,12 +32,13 @@
             let
               pkgs' = pkgs.pkgsCross.aarch64-multiplatform-musl;
             in
-            pkgs'.lib.callPackageWith pkgs' ./nix {
+            (pkgs'.lib.callPackageWith pkgs' ./nix {
               static = true;
               nix-filter = nix-filter.lib;
               crossName = "aarch64";
-            };
-
+            }).overrideAttrs (o: {
+              OCAMLFIND_CONF = pkgs'.ocamlPackages.findlib.makeFindlibConf native o;
+            });
         };
       });
 }

